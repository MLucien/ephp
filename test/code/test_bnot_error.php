<?php

print "Hello world!\n";
$a = ~array();

print "Dark side!\n";

