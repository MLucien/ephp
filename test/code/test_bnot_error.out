Hello world!

Fatal error: Unsupported operand types in {{CWD}}/test/code/test_bnot_error.php on line 4
